accounts.go

```diff
@@ -306,53 +307,58 @@ func (srv *accountsAPI) Authenticate(ctx context.Context, in *accountsv1.Authent
 	return &accountsv1.AuthenticateResponse{Token: tokenString}, nil
 }
 
-func (srv *accountsAPI) AuthenticateGoogle(ctx context.Context, in *accountsv1.AuthenticateGoogleRequest) (*accountsv1.AuthenticateGoogleResponse, error) {
-	err := validators.ValidateAuthenticateGoogleRequest(in)
+func getGoogleUserInfo(accessToken string) ([]byte, error) {
```

Comment:
private function are usually at the end of the files after the Public ones (the endpoints here)